libr/debug/p/debug_native.c

```diff
@@ -548,6 +548,78 @@ static RList *r_debug_native_tids (RDebug *dbg, int pid) {
 	return NULL;
 }
 
+static void r_debug_native_list_proc(RList **list, int pid){
+	DIR *dh;
+	struct dirent *de;
+	char *ptr, st, buf[1024];
+	int i, uid;
+	if (pid) {
+		/* add the requested pid. should we do this? we don't even know if it's valid still.. */
+		r_list_append (*list, r_debug_pid_new ("(current)", pid, 0, 's', 0));
+	}
+	dh = opendir ("/proc");
```

Comment:
use r_util api instead of posix things that dont work on windows for example

Reply:
i know this is not suposed to run on windows. but this code you wrote here is not inside any ifdef , so it will break the windows build